components/ThemedImage.tsx

```
@@ -0,0 +1,44 @@
+export function ThemedImage({
+    style,
+    lightColor,
+    darkColor,
+    variant = 'default',
+    ...rest
+}: ThemedImageProps) {
+    const backgroundColor = useThemeColor({ light: lightColor, dark: darkColor }, 'background');
+
+    const getImageStyle = () => {
+        switch (variant) {
+            case 'success':
```

ESLint

[error] Expected indentation of 8 spaces but found 12.

(@stylistic/js/indent)